engine/src/main.cpp

```diff
@@ -716,162 +717,193 @@ void bootstrap_main_loop() {
         }
     }
+
+    if (cmd_args.count("num-players")) {
+        int num_players = cmd_args["num-players"].as<int>();
+        if (num_players > 1 && num_players <= 9) {
+            CONFIGFILE = new char[42];
+            snprintf(CONFIGFILE, 42, "vegastrike.config.%dplayer", num_players);
+        } else if (num_players != 1) {
+            VS_LOG(warning, "Specified number of players out of range (1-9)");
+        }
+    }
+
+    if (cmd_args.count("mod")) {
+        retstr = cmd_args["mod"].as<std::string>();
+    }
+
+    if (cmd_args.count("start-in-system")) {
+        std::string st;
+        st = cmd_args["start-in-system"].as<std::string>();
+        SetPlayerSystem(st, true);
+    }
+
+    if (cmd_args.count("location")) {
+        QVector PlayerLocation;
+        try {
+            std::string input = cmd_args["location"].as<std::string>();
+            std::istringstream iStringStream(input);
+            iStringStream.ignore(2);
+            iStringStream >> PlayerLocation.i;
+            iStringStream.ignore(1, ',');
+            iStringStream >> PlayerLocation.j;
+            iStringStream.ignore(1, ',');
+            iStringStream >> PlayerLocation.k;
+        } catch (std::ios_base::failure &inputFailure) {
+            std::cout << "Error reading coordinates for player location: " << inputFailure.what()
+                    << std::endl;
+            exit(EXIT_FAILURE);
+        }
+        SetPlayerLoc(PlayerLocation, true);
+    }
+
+    if (cmd_args.count("debug")) {
+        char vs_debug_level_temp = cmd_args["debug"].as<char>();
+        switch (vs_debug_level_temp) {
+            case 0:
+                g_game.vsdebug = 0;
+                break;
+            case 1:
+                g_game.vsdebug = 1;
+                break;
+            case 2:
+                g_game.vsdebug = 2;
+                break;
+            case 3:
+                g_game.vsdebug = 3;
+                break;
+            case '0':
+                g_game.vsdebug = 0;
+                break;
+            case '1':
+                g_game.vsdebug = 1;
+                break;
+            case '2':
+                g_game.vsdebug = 2;
+                break;
+            case '3':
+                g_game.vsdebug = 3;
+                break;
+            default:
+                VS_LOG_FLUSH_EXIT(fatal, "Invalid debug level specified", EXIT_FAILURE);
+        }
+    }
+
+    if (cmd_args.count("net")) {
+        //don't ignore the network section of the config file
+        ignore_network = false;
+    }
+
+    if (cmd_args.count("benchmark")) {
+        benchmark = cmd_args["benchmark"].as<double>();
+    }
+
+    if (cmd_args.count("test-audio")) {
+        // This is handled separately, since we have to undefine main first
+        // return Audio::Test::main(argc, argv);
+    }
+
+    if (cmd_args.count("a")) {
```

Comment:
Delete all three resolutions. We shouldn't support anything under 1920x1080.

Reply:
I disagree about not supporting anything under 1920x1080. There are times -- especially in a VM -- when running with a lower resolution is necessary.

With that said, 800x600 is ridiculously low. I think I will go ahead and remove that option.

Reply:
800x600 option deleted.

Reply:
@royfalk I don't think I've ever run at 1920x1080 - that would be full screen for me. I don't think I run as small as 800x600 often.